sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -78,6 +78,14 @@ public class EventProcessorClient : EventProcessor<EventProcessorPartition>
         ///   a deadlock scenario, especially if using the synchronous form of the call.
         /// </summary>
         ///
+        /// <remarks>
+        ///   This handler will be invoked concurrently, limited to one call per partition.  The processor will await each invocation before starting to process
+        ///   the associated partition.
+        ///
+        ///   No time limit is imposed on an invocation of this handler; the processor will wait indefinitely for execution to complete.  It is recommended for
```

Comment:
nit: the first two sentences seem duplicative. 
```suggestion
        ///   The processor will wait indefinitely for execution of the handler to complete.  It is recommended for
```

Reply:
Applied the same feedback to the base class comment.